modify-and-filter-traffic/conditions-and-actions.md

```diff
@@ -433,52 +436,49 @@ Apart from returning files or predefined responses, a rule can perform the follo
             <td>n/a</td>
             <td>n/a</td>
             <td>Uses the <a href="https://docs.telerik.com/fiddler/knowledge-base/autoresponder#matching-rules">legacy Fiddler Classic string literals and regular expressions.</a></td>
-            <td>Final</td>
+            <td>Non-final</td>
         </tr>
     </tbody>
 </table>
 
 ## Final and Non-Final Actions
 
-Rule actions can be divided into **final** and **non-final** depending on their behavior.
+Rule actions can be divided into **final** and **non-final** depending on their behavior and whether their presence will allow our actions and rules to be executed.
 
 When you work with final and non-final actions, take into consideration the following insights:
 
-* Final actions immediately prevent any other action from executing if this action comes lower on the list for the specific rule. Final actions also prevent any rule with lower priority, which matches the request, from executing any actions.
-
-* Final actions are valid only when the rule is matched during the HTTP request.
-
-* If a session is matched with conditions that depend on its response (for example, a response body contains "HTML"), then any final action in any rule that matches the session will be ignored. The reason for this behavior is that final actions replace the response. By design, Fiddler is not intended to replace a response that was already received and matched conditions in a rule.
-
-* Non-final actions will allow other actions from the same rule or different rules with lower priority, which match the session, to execute.
-
+* Final actions prevent any other action from executing if this action comes lower on the list for the specific rule. 
 
-The following examples demonstrate what will happen when you combine final and non-final actions in one or multiple rules.
+* Final actions prevent any other rule from executing if this rule comes lower on the list of activated rules.
```

Comment:
"if this rule" -> it is unclear whether you speak of the final rule, or of the other rule that comes lower in the rules queue.